nc-news-frontend/src/pages/Articles.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { getAllArticles } from '../api/api';
+
+import ArticleCard from "../components/ArticleCard";
+
+const Articles = () => {
+    const [articles, setArticles] = useState([])
+    const [loading, setLoading] = useState(false);
+
+    useEffect(() => {
+        const fetchArticles = () => {
+          setLoading(true);
```

Comment:
we cna just have the loading state start off life as true then we wouldn't need this line here